packages/client/src/tools.js

```diff
@@ -882,8 +883,20 @@ export default class Tools {
   static extractAbstract(url) {
     const metaExtractionUrl = 'https://meta-extraction-wivlbyt6ga-uc.a.run.app/metadata';
     const queryString = generateQueryString({ url });
-    return fetch(`${metaExtractionUrl}?${queryString}`, {
+    const request = fetch(`${metaExtractionUrl}?${queryString}`, {
       method: 'GET',
     });
+    return request.then(async result => {
+      if (result.status === 200) {
+        return result.json();
+      }
+      const contentType = result.headers.get('content-type');
+      throw new OpenReviewError({
+        name: 'ExtractAbstractError',
+        message: (contentType && contentType.indexOf('application/json') !== -1) ? await result.json() : await result.text(),
```

Comment:
I think you have to get the message value from the json
```suggestion
        message: (contentType && contentType.indexOf('application/json') !== -1) ? (await result.json()).message : await result.text(),
```

Reply:
I wasn't sure if the json has the message property but I can do that.

Reply:
Done.